pkg/virt-handler/device-manager/usb_device.go

```diff
@@ -44,7 +44,7 @@ import (
 	pluginapi "kubevirt.io/kubevirt/pkg/virt-handler/device-manager/deviceplugin/v1beta1"
 )
 
-const (
+var (
```

Comment:
I don't think its good to make this exported const a writable var. Can you change `discoverPluggedUSBDevices` to take the path as a function parameter instead?

Reply:
An alternative is to un-export it.

Reply:
Just to confirm, It was a mistake on my side to have it exported, probably a left over from different interactions in the code...